native/cocos/renderer/gfx-metal/MTLCommandBuffer.mm

```diff
@@ -295,11 +295,13 @@ of this software and associated engine source code (the "Software"), a limited,
     }
 
     Rect scissorArea = renderArea;
-#if defined(CC_DEBUG) && (CC_DEBUG > 0)
     const Vec2 renderTargetSize = ccMtlRenderPass->getRenderTargetSizes()[0];
-    scissorArea.width = MIN(scissorArea.width, renderTargetSize.x - scissorArea.x);
-    scissorArea.height = MIN(scissorArea.height, renderTargetSize.y - scissorArea.y);
-#endif
+    _currentFbWidth = MIN(scissorArea.width, renderTargetSize.x - scissorArea.x);
+    _currentFbHeight = MIN(scissorArea.height, renderTargetSize.y - scissorArea.y);
```

Comment:
In `endRenderPass`, we can set '_currentFbWidth', '_currentFbHeight' to 0. Make it easier to debug.

Reply:
We can also add some `assert(_currentFbWidth == 0)` before assignment to prevent potential size mismatch.